kubernetes/kube.libsonnet

```diff
@@ -739,6 +740,12 @@
     },
   },
 
+  TemporalPortMap(name, bento):: (
+    local ports = if std.objectHas(temporalPorts, name) && std.objectHas(temporalPorts[name], bento) then temporalPorts[name][bento]
+                else { TEMPORAL_FRONTEND: '6933', TEMPORAL_HISTORY: '6934', TEMPORAL_MATCHING: '6935', TEMPORAL_WORKER: '6939',};
```

Comment:
Can we change the defaults so that they do not conflict with anything currently in prod? These ports are the same as DSNG

Reply:
I had a similar concern. The defaults originally were - https://github.com/getoutreach/bootstrap/pull/707/files